src/components/orders/index.js

```diff
@@ -0,0 +1,95 @@
+import _ from 'lodash'
+import React from 'react'
+import * as Redux from 'redux'
+import * as ReactRedux from 'react-redux'
+import * as ReactReduxFirebase from 'react-redux-firebase'
+import moment from 'moment'
+import { withStyles } from '@material-ui/core/styles'
+import PropTypes from 'prop-types'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Avatar from '@material-ui/core/Avatar'
+import ImageIcon from '@material-ui/icons/Image'
+
+import selectors from '../../redux/selectors'
+
+const styles = theme => ({
+  root: {
+    width: '100%',
+    backgroundColor: theme.palette.background.paper,
+  },
+})
```

Comment:
For future, the styles will become pretty big and should be in a separate file:

```js
// components/orders/index.style.js

export default theme => ({
  root: {
    width: '100%',
    backgroundColor: theme.palette.background.paper,
  },
})
```

Not critical for now tho. We can do that when we're implementing design